python/ray/tune/hpo_scheduler.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import copy
+import numpy as np
+try:
+    import hyperopt as hpo
+except Exception as e:
+    print("HyperOpt not found; perhaps not installed?")
```

Comment:
If you want to make this a soft dep, set hpo = None, and check this in the HyperOptScheduler constructor